### DISCUSSION TITLE
Hide Tab Navigation when viewing code?

### DISCUSSION THREAD

**Discussion by homit-dalia** [community/community#52462] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.99.0 (90728769)
Device: iPhone Xʀ (iOS 15.7.1)
</details>

**Comment by stevepopovich** [comment]
Hi @homit-dalia the nav bar should be hidden when you are viewing code. What are you suggesting we should change here?

**Reply by homit-dalia** [reply]
The nav bar is not hidden when I view code. Attaching images for reference. I am now on the latest version of GitHub iOS.

![image](https://user-images.githubusercontent.com/103167599/231491413-c8913724-2ca0-4eb2-803a-266fc92fcd65.png)

![image](https://user-images.githubusercontent.com/103167599/231491594-bc193b96-bc50-40e2-a24d-26dc1664c0d1.png)

**Reply by stevepopovich** [reply, marked as answer]
Ok, thanks for the feedback. I passed this onto the team for prioritization

**Reply by homit-dalia** [reply]
It’s been 2 months. The issue has not been fixed.

**Reply by stevepopovich** [reply]
This is just not a feature we have prioritized at this time. We will keep the feedback in mind though.

**Reply by verhovsky** [reply]
Please implement this, it hides about 5/21 lines in landscape mode on my iphone 15 pro.

Either just tapping the screen anywhere or swiping up on the filename.
